### PR TITLE
Create script to populate with fake data db

### PR DESCRIPTION
**Pull request by caxtonacollins**
Created script to populate db with dummy data

### CODE REVIEW

File: web_app/db/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Populating the Database with Fake Data
+
+To populate the database with fake data for testing purposes, follow these steps:
+
+1. **Install Dependencies**: Ensure you have the required packages installed. You can do this by running:
+
+   ```bash
+   pip install sqlalchemy psycopg2 faker
```
Comment (djeck1432) [inline web_app/db/README.md:8]: we don't use on the project `pip`, please add these dependencies add to pyproject.toml 
most of them already there

File: web_app/db/seed_data.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import os 
+import sys
+from sqlalchemy import create_engine  
+from sqlalchemy.orm import sessionmaker  
+from faker import Faker  
+# from web_app.db.models import User, Position, AirDrop, TelegramUser
```
Comment (djeck1432) [inline web_app/db/seed_data.py:6]: remove unused imports

File: web_app/db/seed_data.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+DB_NAME = os.getenv("DB_NAME")
+DB_HOST = os.getenv("DB_HOST")
+DB_PORT = os.getenv("DB_PORT")
+
+DATABASE_URL = "postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
+
+# Create a new database session  
+engine = create_engine(DATABASE_URL)  
+Session = sessionmaker(bind=engine)  
+session = Session()
```
Comment (djeck1432) [inline web_app/db/seed_data.py:27]: REmove this code:
```
# Database connection settings  
DB_USER = os.getenv("DB_USER")
DB_PASSWORD = os.getenv("DB_PASSWORD")
DB_NAME = os.getenv("DB_NAME")
DB_HOST = os.getenv("DB_HOST")
DB_PORT = os.getenv("DB_PORT")

DATABASE_URL = "postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"

# Create a new database session  
engine = create_engine(DATABASE_URL)  
Session = sessionmaker(bind=engine)  
session = Session() 
```
import it from [here](https://github.com/djeck1432/spotnet/blob/main/web_app/db/database.py#L30)

File: web_app/db/seed_data.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+# Create a new database session  
+engine = create_engine(DATABASE_URL)  
+Session = sessionmaker(bind=engine)  
+session = Session()  
+
+# Initialize Faker  
+fake = Faker()  
+
+def create_users():
```
Comment (djeck1432) [inline web_app/db/seed_data.py:32]: add docstring and annotation types

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Fix pylint issues:
```
pylint $(git ls-files '*.py') --disable=all --enable=C0114,C0115,C0116,C0301
```
run this command to see them

File: web_app/db/README.md (hunk 1 above)
Comment (caxtonacollins) [reply web_app/db/README.md:8]: okay sir

**Comment by caxtonacollins** [comment]
@djeck1432 please review

**Comment by caxtonacollins** [comment]
Screenshot 2024-10-31 134608

@djeck1432 please review

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**